src/all.js

```diff
@@ -12,6 +13,12 @@ function initAll () {
   // Find all buttons with [role=button] on the document to enhance.
   new Button(document).init()
 
+  // Find all global accordion components to enhance.
+  var $accordions = document.querySelectorAll('[data-module="accordion"]')
```

Comment:
Should we not be name spacing data attributes? eg. `data-govuk-module`? Perhaps something to look at as a separate PR

Reply:
Yes good point. We don't currently namespace data attributes in any of our components but it's been discussed before. I've added a card to Dev Time to discuss again as a team.